docs/reference/changelog.rst

```diff
@@ -2,15 +2,22 @@
 Changelog
 *********
 
+2.5.1 (2025-02-11)
```

Comment:
@mr-cal are you ok with doing a release today?

Reply:
Oh - sorry I should have asked before I made this changelog. Please let me know your preferred release date and I'll adjust the dates accordingly!

Reply:
Yes, I can release today. If it's not ready until tomorrow, @tigarmo can release it.